worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -452,6 +457,23 @@ namespace RTC
 			return;
 		}
 
+		// Check if RTP packet is too old to be stored.
+		if (this->storageItemBuffer.GetBufferSize() > 0)
+		{
+			auto* storageItem = this->storageItemBuffer.GetFirst();
+
+			// Processing RTP packet is older than first one.
+			if (RTC::SeqManager<uint32_t>::IsSeqLowerThan(packet->GetTimestamp(), storageItem->timestamp))
+			{
+				uint32_t diffTs{ storageItem->timestamp - packet->GetTimestamp() };
+
+				// RTP packet is older than the retransmission buffer size.
+				if (static_cast<uint32_t>(diffTs * 1000 / this->params.clockRate) >= this->retransmissionBufferSize)
+
```

Comment:
Annoying blank line or missing brackets?

Reply:
The former, fixed.